### PR TITLE
Clean up CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by aeolianeth**
This commit makes minor structural changes to the CONTRIBUTING.md file.

These changes aim to make this file more readable for new contributors, improving accessibility.

### CODE REVIEW

**Comment by aeolianeth** [comment]
Hi @peripheralist 👋 , this is my first contribution. There'll be more to come!

Would you mind reviewing this PR when you have a chance? 🙇

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
-- Be approved by at least one code owner
+Start with issues labelled
+[`good first issue`](https://github.com/jbx-protocol/juice-juicehouse/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
 
-## Finding a first issue
+### Create a pull request
 
-Start with issues with the label
-[`good first issue`](https://github.com/jbx-protocol/juice-juicehouse/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+To contribute, create a pull request (PR) that tartgets the `main` branch. A
```
Comment (peripheralist) [inline CONTRIBUTING.md:20]: little typo! tartgets

**Comment by peripheralist** [comment]
thanks for you help @aeolianeth! good improvements. let's fix that little typo and we can merge

File: CONTRIBUTING.md (hunk 1 above)
Comment (aeolianeth) [reply CONTRIBUTING.md:20]:
```suggestion
To contribute, create a pull request (PR) that targets the `main` branch. A
```

**Comment by aeolianeth** [comment]
@peripheralist thanks for the review and good catch on the typo 🙈 . Back to you 🏓

**Review by peripheralist — APPROVED** [review]

**Pull request merged**